CHANGES.md

```diff
@@ -3,19 +3,101 @@
 Rust image aims to be a pure-Rust implementation of various popular image formats. Accompanying reading/write support, rust image provides basic imaging processing function. See `README.md` for further details.
 
 ## Known issues
- - Images with *n* bit/channel (*n ≠ 8*) are not well supported but basic
-     support for 16-bit is available and implemented for PNG.
  - Not all Interlaced (progressive) or animated images are well supported.
  - The color space information of pixels is not clearly communicated.
+ - Initialization via byte-based buffers and ffi is a work-in-progress.
 
 ## Changes
 
```

Comment:
Could you add an `### Unreleased` heading here so we can try to keep up to date with changes instead of having to write a whole changelog right before the release?

Reply:
I'll actually do the proper thing and rebase on another branch of fixes. Then make this small adjustment to the file and then do the commit today. Fine?